project/StacksafeScalapbGenerator.scala

```diff
@@ -220,27 +189,26 @@ class StacksafeProtobufGenerator(params: GeneratorParams) extends ProtobufGenera
       }
       .print(message.getOneofs.asScala)(
         (printer, oneof) =>
-          printer.add(s"var __${oneof.scalaName} = this.${oneof.scalaName.asSymbol}")
+          printer.add(s"var __${oneof.scalaName.name} = this.${oneof.scalaName.nameSymbol}")
       )
-      .addStringMargin(s"""var _done__ = false
-           |
-           |Sync[F].whileM_ (Sync[F].delay { !_done__ }) {
-           |  for {
-           |    _tag__ <- Sync[F].delay { _input__.readTag() }
-           |    _ <- _tag__ match {
-           |      case 0 => Sync[F].delay { _done__ = true }""")
+      .add(s"""var _done__ = false
+              |
+              |Sync[F].whileM_ (Sync[F].delay { !_done__ }) {
+              |  for {
+              |    _tag__ <- Sync[F].delay { _input__.readTag() }
+              |    _ <- _tag__ match {
+              |      case 0 => Sync[F].delay { _done__ = true }""".stripMargin)
       .print(message.fields) { (printer, field) =>
         val p = {
-
           val newValBaseF = if (field.isMessage) {
             val defInstance =
-              s"${field.getMessageType.scalaTypeNameWithMaybeRoot(message)}.defaultInstance"
+              s"${field.getMessageType.scalaType.fullNameWithMaybeRoot(message)}.defaultInstance"
             val baseInstance =
               if (field.isRepeated) defInstance
               else {
                 val expr =
                   if (field.isInOneof)
-                    fieldAccessorSymbol(field)
+                    s"this.${fieldAccessorSymbol(field)}"
```

Comment:
why changing here?

Reply:
Yes, nothing changed here, I rolled it back.